operations/services.md

```diff
@@ -873,24 +879,24 @@ Creates a new order with the specified products and items. Only positive charges
 | --- | --- | --- | --- |
 | `ProductId` | string | required | Unique identifier of the [Product](services.md#product) to be ordered. |
 | `Count` | number | optional | Count of products to be ordered, e.g. 10 in case of 10 beers. |
-| `UnitCost` | [Cost](services.md#cost) | optional | Unit cost of the product that overrides the cost defined in Mews. |
+| `UnitAmount` | [Amount](services.md#amount) | optional | Unit amount of the product that overrides the cost defined in Mews. |
 
 #### Item parameters
 
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
 | `Name` | string | required | Name of the item. |
 | `UnitCount` | number | required | Count of units to be ordered, e.g. 10 in case of 10 beers. |
-| `UnitCost` | [Cost](services.md#cost) | required | Unit cost, e.g. cost for one beer \(note that total cost of the item is therefore `UnitCount` times `UnitCost`\). |
+| `UnitAmount` | [Amount](services.md#amount) | required | Unit amount, e.g. amount for one beer \(note that total amount of the item is therefore `UnitAmount` times `UnitAmount`\). |
 | `AccountingCategoryId` | string | optional | Unique identifier of an [Accounting category](finance.md#accounting-category) to be assigned to the item. |
 
-#### Cost
+#### Amount parameters
 
 | Property | Type |  | Description |
 | --- | --- | --- | --- |
-| `Amount` | decimal | required | Amount including tax. |
+| `GrossValue` | decimal | required | Amount including tax. |
 | `Currency` | string | required | ISO-4217 code of the [Currency](configuration.md#currency). |
-| `Tax` | decimal | required | Tax rate, e.g. `0.21` in case of 21% tax rate. |
+| `TaxCodes` | array of string [Tax Codes](configuration.md#tax-rates) | required | Tax codes to be applied to the item. |
```

Comment:
Missing information about how many taxes you can include in the net or gross pricing.
For gross pricing:
Gross value and 1 tax
Net value and multiple taxes